### PR TITLE
SA region removed from TEST and SAND

### PR DESCRIPTION
**Pull request by VaclavK**

### CODE REVIEW

File: src/Eshopworld.DevOps/Eshopworld.DevOps.csproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>4.0.0</Version>
-    <PackageReleaseNotes>V2 region list sync</PackageReleaseNotes>
+    <Version>5.0.0</Version>
```
Comment (arturek) [inline src/Eshopworld.DevOps/Eshopworld.DevOps.csproj]: 4.1 should be enough. It's not a breaking change. And nobody should even noticed it.

Comment (dguerin) [reply src/Eshopworld.DevOps/Eshopworld.DevOps.csproj]: if someone is using it, it would be a breaking change.

Comment (dguerin) [reply src/Eshopworld.DevOps/Eshopworld.DevOps.csproj]: After some clarity from @arturek I tend to agree, It's not a breaking change per say, the list is always returned with a list of entries, SA just won't be in it anymore.

Comment (VaclavK) [reply src/Eshopworld.DevOps/Eshopworld.DevOps.csproj]: as mentioned on the call, the only usage I am aware of is the cosmos preferred region list so even if they run in SA for little while longer, not going to SA (closest) will have some but not considerable performance implications...

**Pull request merged**